.eslintrc

```diff
@@ -1,9 +1,11 @@
 {
-    "parser": "babel-eslint",
+    "parser": "@typescript-eslint/parser",
```

Comment:
🗒️ This lets eslint parse and lint the typescript files. Also underneath it adds some typescript specific eslint rules